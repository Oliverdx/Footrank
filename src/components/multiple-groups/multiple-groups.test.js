import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import MultipleGroups from './multiple-groups';

describe('Tests of Multiple groups Component', () => {

  it('Prevent the component to not render if not receive props', () => {
    const component = renderer
      .create(<MultipleGroups />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });

  it('Have to return two groups only with the title', () => {
    const mock = [{ group: 'GROUP A' }, { group: 'GROUP B' }]

    const component = mount(<MultipleGroups standings={mock} />);

    expect(component.find('h3').length).toEqual(2);
  });

});


