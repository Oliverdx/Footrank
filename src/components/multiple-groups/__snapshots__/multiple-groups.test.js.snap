// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests of Multiple groups Component Prevent the component to not render if not receive props 1`] = `null`;
